README.md

```diff
@@ -73,7 +74,7 @@ Wait for cluster to be created:
 > When ready, status will change from `PROVISIONING` to `RUNNING`
 
 ```shell
-gcloud container clusters list --filter="resourceLabels.demo:build"
+watch gcloud container clusters list --filter="resourceLabels.demo:build"
```

Comment:
That assumes watch on the system, no? Should this be `--watch` flag in `kubectl`?

Reply:
We may just want to keep it the way it is then because Im not sure a kubectl command could show the status of two clusters, but I could be wrong.

Reply:
That file is an output by the config init so really shouldn't be pushed back to the repo. I didn't include in the .gitignore to make sure people can see the final product but I guess this will cause issues.